ow_lander/src/ow_lander/constants.py

```diff
@@ -58,7 +58,10 @@
 # Distance between scoop center of mass and lower blade
 SCOOP_HEIGHT = 0.076
 
-DEFAULT_GROUND_HEIGHT = -0.155
+# selected by using ArmFindSurface to interrogate several spots directly in
+# front of the lander in both atacama_y1a and europa_terminator_workspace and
+# averaging the results
+DEFAULT_GROUND_HEIGHT = -0.172
```

Comment:
Since you showed an interest in this, I thought I would share the following. In atacama_y1a the terrain height is about -0.18, and in europa_terminator_workspace it's about -0.16. So this is not really correct for either workspace. Perhaps this is just a default for something that should not be defaulted?

Reply:
Now that you mention it, I think it best not to default ground height.  It's been a "convenience" that is fine for our daily purposes, but conceptually should not be a default.

Here's an idea for an extension of this convenience, if feasible.  Maintain a mapping of our provided terrains (with default lander location) to some suitable ground height, if this can be easily looked up, by the arm actions as well as autonomy.

Reply:
> Here's an idea for an extension of this convenience, if feasible. Maintain a mapping of our provided terrains (with default lander location) to some suitable ground height, if this can be easily looked up, by the arm actions as well as autonomy.

If the info was readily available to the lander, I would be all for this, but at the moment I do not know how to make lander actions aware of the name of the terrain.

Reply:
The default values are also not a part of the action server because ROS Actions do not support default parameters. They instead only exist inside their corresponding action clients. So whatever operation it did to lookup the current terrain, it would have to do each call of an action client since those are not persistent.

Reply:
I want to retract my suggestion of maintaining maps to default ground locations because another modeling goal we should aim for is easily altering the placement of the lander on any terrain (Jira to come).  

I assume that a default ground height is useful only for argument-free running of the action client scripts.  It's never something that should be used in autonomy or a mission simulation.  To that end, hardcoding a default that works for our two most popular worlds seems fine for now, if we document that the default invocation may not work on other worlds.